.binder/runtime.txt

```diff
@@ -1 +1 @@
-python-3.9
+python - 3.9
```

Comment:
Note that this *might* break the parsing of the python version (not sure and haven't checked further than the related [doc](https://mybinder.readthedocs.io/en/latest/using/config_files.html#runtime-txt-specifying-runtimes) which does not include spaces around the "-"). 
We should check that binder is working as expected before merging.

Reply:
I can confirm it still works. You can acess the artificats of this PR from the `host_and_deploy_doc` CI job and launch binder from one of the examples from there. A direct link you can test: https://output.circle-artifacts.com/output/job/ea13e23d-6ec6-427e-8d0d-0c99fced5c9b/artifacts/0/dev/auto_examples/00_tutorials/plot_python_101.html#sphx-glr-auto-examples-00-tutorials-plot-python-101-py

Reply:
Awesome ! Thanks for checking !

Reply:
Oh thanks.
Actually I am unusure why this got changed.